lib/broccoli/modules-from-meta.js

```diff
@@ -0,0 +1,15 @@
+/* jshint ignore:start */
+
+Ember['default'].$.unique(Ember['default'].$('meta[data-module=true], meta[name="' + prefix + '/config/environment"]')).each(function(index, meta) {
```

Comment:
`meta[name="' + prefix + '/config/environment"]` and therefore `Ember['default'].$.unique` seem unnecessary. Any time you're using this version of ember-cli you're guaranteed `data-module=true`.